app/controllers/messages.js

```diff
@@ -123,13 +123,25 @@ class MessagesController extends BaseController {
 
     const userLogin = (await User.findOne({ _id: currentUserId }))?.params
       ?.login;
+    const titleImgUrl = !messageParams.attachments?.length
```

Comment:
let's call this field `firstAttachmentUrl `

And also we need to add this into documentation